client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx

```diff
@@ -92,6 +93,24 @@ function createSeries( series: Array< SeriesProp > ) {
 	return [ { label: ' ' }, ...configuredSeries ];
 }
 
+function addExtraScaleIfDefined( series: Array< SeriesProp > ) {
+	const scale = series.find( ( serie ) => serie.scale )?.scale;
+	if ( scale ) {
+		return [
```

Comment:
What do you think about using early return here?

Reply:
I'm not sure I understand the suggestion. Can you clarify?

Reply:
I think it's cleaner to use an 'early return' approach if possible:

```
if ( ! scale ) {
    return [];
}
// the rest of the code
```